website/config/routes.js

```
@@ -781,9 +781,10 @@ module.exports.routes = {
   'GET /handbook/marketing/docs-handbook/': '/handbook/company/communications#docs',
   'GET /handbook/marketing/website-handbook/': '/handbook/company/communications#website',
   'GET /handbook/quality': '/handbook/engineering#quality',
-  'GET /device-management/fleet-user-stories-f100': '/success-stories/fleet-user-stories-wayfair',
-  'GET /device-management/fleet-user-stories-schrodinger': '/success-stories/fleet-user-stories-wayfair',
-  'GET /device-management/fleet-user-stories-wayfair': '/success-stories/fleet-user-stories-wayfair',
+  'GET /success-stories': '/customers',
```

ESLint

[error] Duplicate key 'GET /success-stories'.

(no-dupe-keys)